docs/architecture.md

```diff
@@ -8,209 +8,175 @@
 
 Taking into account that [web3.storage](http://web3.storage) onboards any type of content (up to a maximum of 4GiB-padded shards to have better utilization of Fil sector space), multiple CAR files uploaded need to be aggregated into a bigger Piece that can be offered to Filecoin Storage Providers. w3filecoin pipeline keeps track of queued CARs (cargo) to be included in Storage Provider deals.
```

Comment:
> Currently, Spade requires a Filecoin Piece with size between 15.875GiB and 31.75GiB to create deals with Filecoin Storage Providers. Moreover, the closer a Filecoin Piece is closer to the upper bound, the most optimal are the associated storage costs.

This is not changed here, but is that accurate ? Piece sizes should be pow of 2, so it 16GiB to 32GiB would be the pieces sizes. Perhaps it was referring to unpadded sizes which I think would come to those numbers, however talking in pre-padded sizes while we submit pieces seems confusing.